src/app/+item-page/edit-item-page/item-relationships/edit-relationship-list/edit-relationship-list.component.html

```diff
@@ -0,0 +1,15 @@
+<ng-container *ngVar="(updates$ | async) as updates">
+  <div *ngIf="updates">
+    <h5>{{getRelationshipMessageKey(relationshipLabel) | translate}}</h5>
+    <ng-container *ngVar="(updates | dsObjectValues) as updateValues">
```

Comment:
Hi @paulo-graca
I would like to handle that use case in a separate PR as it will be dependent on https://github.com/DSpace/DSpace/pull/2450

Once that's in master, we can use paging per relationship type, which would be important for this functionality. This can also be used for item display pages once merged